### ISSUE TITLE
small object detection 

### ISSUE THREAD

**Issue by zxq309** [ultralytics/yolov5#6981]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

This paper {A Normalized Gaussian Wasserstein Distance for Tiny Object Detection[https://arxiv.org/abs/2110.13389], github[https://github.com/bhyun-kim/NWD]}
replaces IOU computational loss to help small object detection based Faster R-CNN，Do you have plans to add to the code？

**Comment by glenn-jocher** [comment]
@XiangqingZhang hi, thank you for your feature suggestion on how to improve YOLOv5 🚀! See ICIoU PR https://github.com/ultralytics/yolov5/pull/6977 for an alternative attempt at box regression improvement. 

The fastest and easiest way to incorporate your ideas into the official codebase is to submit a Pull Request (PR) implementing your idea, and if applicable providing before and after profiling/inference/training results to help us understand the improvement your feature provides. This allows us to directly see the changes in the code and to understand how they affect workflows and performance.

Please see our ✅ [Contributing Guide](https://docs.ultralytics.com/help/contributing/) to get started.

**Comment by glenn-jocher** [comment]
@XiangqingZhang I reviewed the paper. Seems interesting, but no results on standard datasets like VOC/COCO, and no code implementation either = difficult to reproduce.

**Comment by glenn-jocher** [comment]
@XiangqingZhang I put a help wanted tag on this to see if anyone wants to contribute. The idea seems interesting. Perhaps it's worthwhile contacting the authors to see if they can assist with an implementation or even just a PyTorch loss function to get us started.

**Comment by zxq309** [comment]
@glenn-jocher the authors give mmdetection code: https://github.com/bhyun-kim/NWD/blob/main/Tiny_Object_Detection_Normalized_Gaussian_Wasserstein_Distance.ipynb, you can see it.

**Comment by glenn-jocher** [comment]
@XiangqingZhang great! I don't have time to experiment myself, but you can introduce modifications into the IoU function here to test:
https://github.com/ultralytics/yolov5/blob/c09fb2aa95b6ca86c460aa106e2308805649feb9/utils/metrics.py#L202-L242

**Comment by LYMDLUT** [comment]
> I am focus on this problem,could you contact me and we can talk about it on wechat ,13009275407
